apps/finance/app/src/components/Transfers.js

```diff
@@ -58,6 +89,47 @@ class Transfers extends React.PureComponent {
       ...initialState,
     })
   }
+  handleDateRangeChange = selectedDateRange => {
+    this.setState({ selectedDateRange })
+  }
+  encodeDataToCsv = (data, tokenDetails) => {
+    const csvContent = [
+      'data:text/csv;charset=utf-8,Date,Source/Recipient,Reference,Amount',
+    ]
+      .concat(
+        data.map(
+          ({
+            date,
+            numData: { amount },
+            reference,
+            isIncoming,
+            entity,
+            token,
+          }) => {
+            const { symbol, decimals } = tokenDetails[toChecksumAddress(token)]
+            const formattedAmount = formatTokenAmount(
+              amount,
+              isIncoming,
+              decimals,
+              true,
+              { rounding: 5 }
+            )
+            return `${format(
+              date,
+              'dd/MM/yy'
+            )},${entity},${reference},${`${formattedAmount} ${symbol}`}`
+          }
+        )
+      )
+      .join('\n')
+    return window.encodeURI(csvContent)
+  }
+  getCsvFilename = () => {
+    const { selectedDateRange } = this.state
+    const start = format(selectedDateRange.start, 'MMM dd yyyy')
+    const end = format(selectedDateRange.end, 'MMM dd yyyy')
+    return `Finance (${start} to ${end}).csv`
```

Comment:
For consistency with https://github.com/aragon/aragon/pull/656#discussion_r269028909 

I'd suggest avoiding spaces and using the standard `yyyy-mm-dd` format